app/_data/docs_nav_gateway_3.0.x.yml

```diff
@@ -74,117 +68,131 @@ items:
-        url: /kong-production/kong-openresty
-      - text: Serving a website and API's from Kong
-        url: /kong-production/website-api-serving
+        url: /production-deployment/kong-openresty
+      - text: Serving a Website and API's from Kong
+        url: /production-deployment/website-api-serving
       - text: Monitoring
         items:
           - text: Introduction
-            url: /kong-production/monitoring/
-          - text: How to monitor with Prometheus
-            url: /kong-production/monitoring/prometheus
-          - text: How to monitor with Statsd
-            url: /kong-production/monitoring/statsd
+            url: /production-deployment/monitoring/
+          - text: How to Monitor with Prometheus
+            url: /production-deployment/monitoring/prometheus
+          - text: How to Monitor with Statsd
+            url: /production-deployment/monitoring/statsd
           - text: Collect Metrics with Datadog
-            url: /kong-production/monitoring/datadog
+            url: /production-deployment/monitoring/datadog
       - text: Tracing
         items:
           - text: Overview
-            url: /kong-production/tracing/
+            url: /production-deployment/tracing/
           - text: How to Write a Custom Trace Exporter
-            url: /kong-production/tracing/write-custom-trace-exporter
+            url: /production-deployment/tracing/write-custom-trace-exporter
           - text: Tracing API reference
-            url: /kong-production/tracing/api
+            url: /production-deployment/tracing/api
       - text: Resource Sizing Guidelines
-        url: /kong-production/sizing-guidelines
-      - text: Kong security update process
-        url: /kong-production/kong-security-update-process
-      - text: Blue-green deployments
-        url: /kong-production/blue-green
+        url: /production-deployment/sizing-guidelines
+      - text: Security Update Process
+        url: /production-deployment/security-update-process
+      - text: Blue-Green Deployments
+        url: /production-deployment/blue-green
       - text: Canary Deployments
-        url: /kong-production/canary
+        url: /production-deployment/canary
+      - text: Clustering Reference
+        url: /production-deployment/clustering
       - text: Logging Reference
-        url: /kong-production/logging
+        url: /production-deployment/logging
       - text: Upgrade Kong Gateway
-        url: /kong-production/upgrade-enterprise
-      - text: Upgrade Kong Gateway OSS
-        url: /kong-production/upgrade-oss
-      - text: Migrate from OSS to EE
-        url: /kong-production/migrate-ce-to-ke
+        items:
+          - text: Upgrade Kong Enterprise
+            url: /production-deployment/upgrade/enterprise
+          - text: Upgrade Kong Gateway OSS
+            url: /production-deployment/upgrade/oss
+          - text: Migrate from OSS to EE
+            url: /production-deployment/upgrade/migrate-ce-to-ke
```

Comment:
This upgrade section might be worth moving up in the directory structure as well, eg `/upgrade/enterprise` instead of `/production-deployment/upgrade/enterprise`. Nav location would stay the same though.

Reply:
No problems, I just didn't have time to implement this before the end of my day. I'll do it on Monday morning unless you get to it first.

Reply:
Done